airflow/jobs/scheduler_job_runner.py

```diff
@@ -744,6 +746,11 @@ def _process_executor_events(self, session: Session) -> int:
                 ti.queued_by_job_id,
                 ti.pid,
             )
+            arbitrary_log_message = (
+                f"task finished with ti.state={ti.state} state={state} info={info} try={ti.try_number} "
+                f"_try={ti._try_number}"
+            )
+            self._task_log_shipper.info(ti, arbitrary_log_message, self.log)
```

Comment:
As a general theme I would prefer this to use arguments similar to the regular logging function and pass the log parameters separately, instead of formatting the log message eagerly.